src/helpers.rs

```diff
@@ -1196,6 +1196,21 @@ where
     throw_ub_format!("incorrect number of arguments: got {}, expected {}", args.len(), N)
 }
 
+/// Check that the number of args is at least the minumim what we expect
+pub fn check_min_arg_count<'a, 'tcx, const N: usize>(
+    name: &'a str,
+    args: &'a [OpTy<'tcx>],
+) -> InterpResult<'tcx, &'a [OpTy<'tcx>; N]> {
+    if let Some((ops, _)) = args.split_first_chunk() {
```

Comment:
Would be really nice to get this helper in and start using it for all our variadic shims. :)

Reply:
Maybe it'd make sense to split that into a separate PR, to avoid it being held up by getting the Android thing right?

OTOH I think this PR is fairly close to done.

Reply:
Yeah, that's a good idea.